src/main/java/Panna.java

```diff
@@ -0,0 +1,161 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+// to add: tag for A-Classes
+// to add: tag for A-Collections
+public class Panna {
+
+    private static String command = "";
+
+
+
+    public static void main(String[] args) throws PannaException {
+
+        ArrayList<Task> tasks = new ArrayList<>();
+
+        System.out.println("----------------------------------------------------------\n"
+        + "Hello! I'm Panna.\n" +
+                "What can I do for you?\n\n" +
+                "----------------------------------------------------------");
+
+
+        Scanner s = new Scanner(System.in);
+
+        Panna.command = s.nextLine();
+
+        while (!Panna.command.equals("bye")) {
+
+
+
+
+
+            if (Panna.command.equals("list")) {
```

Comment:
Perhaps you could replace the else if statements with a switch statement?

Reply:
Seconding this, using a switch statement would also help with the level of nesting that we have here!